test/commandLine/commandLineTest.js

```diff
@@ -0,0 +1,664 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const rimraf = require('rimraf')
+const fork = require('child_process').fork
+
+describe('Command Line Tests', function () {
+  const appDir = path.join(__dirname, '../app/commandLineTest')
+
+  before(function () {
+    fse.ensureDirSync(path.join(appDir))
+    let dataIn = `const path = require('path')
+    const appDir = path.join(__dirname, '../app/commandLineTest')
+    const app = require('../../../roosevelt')({
+      appDir: appDir,
+      generateFolderStructure: false,
+    })
+    if (process.send) {
+      process.send(app.expressApp.get('params'))
+    }
+
+    process.exit()
+    `
```

Comment:
i don't think we should write this code inline like this. it's pretty hard to read and maintain. put it in a separate file somewhere appropriate in the `test` directory somewhere, then open the file here and dump its contents into the `writeFileSync`